pvlib/temperature.py

```diff
@@ -0,0 +1,253 @@
+"""
+The ``temperature`` module contains functions for modeling temperature of
+PV modules and cells.
+"""
+
+import numpy as np
+
+
+TEMPERATURE_MODEL_PARAMETERS = {
+    'sapm': {
+        'open_rack_glass_glass': {'a': -3.47, 'b': -.0594, 'deltaT': 3},
+        'close_mount_glass_glass': {'a': -2.98, 'b': -.0471, 'deltaT': 1},
+        'open_rack_glass_polymer': {'a': -3.56, 'b': -.0750, 'deltaT': 3},
+        'insulated_back_glass_polymer': {'a': -2.81, 'b': -.0455, 'deltaT': 0},
+    },
+    'pvsyst': {'freestanding': {'u_c': 29.0, 'u_v': 0},
+               'insulated': {'u_c': 15.0, 'u_v': 0}}
+}
+
+
+def _temperature_model_params(model, parameter_set):
+    try:
+        params = TEMPERATURE_MODEL_PARAMETERS[model]
+        return params[parameter_set]
+    except KeyError:
+        msg = ('{} is not a named set of parameters for the {} cell'
+               ' temperature model.'
+               ' See pvlib.celltemp.TEMPERATURE_MODEL_PARAMETERS'
+               ' for names'.format(parameter_set, model))
+        raise KeyError(msg)
+
+
+def sapm_cell(poa_global, temp_air, wind_speed, a, b, deltaT,
+              irrad_ref=1000):
+    r'''
+    Calculate cell temperature per the Sandia PV Array Performance Model [1].
+
+    Parameters
+    ----------
+    poa_global : numeric or Series
```

Comment:
numeric implies Series is allowed, so numeric is the proper description here and the parameters below. See https://pvlib-python.readthedocs.io/en/stable/contributing.html#documentation

See also #765 for changing this nomenclature